src/app/topic/topic.component.ts

```diff
@@ -90,6 +93,18 @@ export class TopicComponent extends RouterEnter {
     }
     this._getTopic();
     this._getTopicProgress();
+    // convert other brand video players to custom player.
+    setTimeout(() => {
+      Array.from(this.document.querySelectorAll('#video')).map(player => {
+        // tslint:disable-next-line:no-unused-expression
+        new Plyr(player as HTMLElement, {ratio: '16:9'});
+        player.classList.add('topic-video');
```

Comment:
Why can't we just add these class names into the HTML directly?

Reply:
main reason is plyr plugin replacing all css classes when it render the player.
if we adding css class directly from html, we need to add that to this element `<div class="plyr__video-embed" id="video" [innerHtml]="iframeHtml"></div>` but when we create player from this line `new Plyr(player as HTMLElement, {ratio: '16:9'});` plugin is replacing all css class that element have. so our class will removed my plugin.
another reason is if we add css class in html we can't add then to embedded videos.